.github/workflows/issue-notifications.yml

```diff
@@ -0,0 +1,19 @@
+name: Issue Notifications
+
+on:
+  workflow_dispatch:
+  issues:
+    types: [opened, edited]
+  issue_comment:
+    types: [created]
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  notify:
+    uses: revenuecat/sdk-github-workflows/.github/workflows/issue-notifications.yml@v1
```

Comment:
Do we need to pass the repo name as a parameter? Or does it pick it up automatically? Honestly now sure where `repo_name` is coming from here: https://github.com/RevenueCat/sdk-github-workflows/blob/main/.github/workflows/issue-notifications.yml